### PR TITLE
Support comma seperator for amount field, dynamically change to dot. 

### LINKED ISSUE #2951: Transfer: amount field should also accept comma as decimal separator
If the user types 0,01 in the amount field, it should be accepted as valid, or the field should replace the comma with a dot (changing it to 0.01).

This is useful for languages that use comma as a decimal separator, specially when typing the value using the numeric keypad.

### PR DESCRIPTION
**Pull request by xmrdsc**
Fixes #2951

### CODE REVIEW

File: pages/Transfer.qml [hunk 1]
```diff
@@ -311,7 +312,7 @@ Rectangle {
                   }
 
                   validator: RegExpValidator {
-                      regExp: /^(\d{1,8})?([\.]\d{1,12})?$/
+                      regExp: /^(\d{1,8})?([\.|,]\d{1,12})?$/
```
Comment (xiphon) [inline pages/Transfer.qml:315]: Treats `1|1` as a valid amount

Comment (xmrdsc) [reply pages/Transfer.qml]: Thanks, my regex sucks, please check my fix.

Comment (xiphon) [reply pages/Transfer.qml]: Should be `^(\d{1,8})?([\.,]\d{1,12})?$`

Comment (xmrdsc) [reply pages/Transfer.qml]: Ehh, yes. Thanks.

**Pull request merged**